Add files via upload

drewm8080 opened the pull request:
Sorry. Forgot to fix something in the last pull request. Use this one.

drewm8080 committed:
Add files via upload

Sorry. Forgot to fix something in the last pull request. Use this one.

Miao-Long commented:
Just write a new commit to the pull request next time. You don't need to open a new PR with every edit.

c-myau commented on timeserieskmeans_final.py:
```diff
@@ -0,0 +1,70 @@
+  plt.rcParams['lines.linewidth'] = 2 
+  plt.rcParams['lines.markersize'] = 8
+  data_c = df.copy()
+  data_c['Class']=y
```
If we have this many parameters, pass in a JSON object or a dictionary or something into the function and define these at script runtime.

c-myau commented on timeserieskmeans_final.py:
```diff
@@ -0,0 +1,70 @@
+  df['Day'] = df.index +1
+
+
+  #assuming first column is the datetime column
```
Jae has an example CSV on google drive but regardless, all we are doing is importing the CSV directly into a dataframe with read_csv(). Use the timeseries column from that

c-myau commented on timeserieskmeans_final.py:
```diff
@@ -0,0 +1,70 @@
+  plt.scatter(centroids[:,0] , centroids[:,1] , s = 250, color = 'k',alpha = 0.9)
+
+  dtw_score = dtw(x, y)
+  print("The DTW score is", dtw_score)
```
Include a [if __name__ == __main__ ... section along with the sample csv data used so we can test this script out as a standalone.

c-myau commented on timeserieskmeans_final.py:
```diff
@@ -0,0 +1,70 @@
+from tslearn.metrics import soft_dtw
+from tslearn.clustering import TimeSeriesKMeans
+
+def kmeans(csv,k,k_seeds,xaxis, yaxis, x_lower=None,x_upper=None,y_lower=None,y_upper=None):
```
Where is the scale parameter for X or Y? We need to be able to adjust the magnitude of the X or Y axis at runtime. Ideally, pass in a mathematical function as a parameter.

drewm8080 committed:
Update timeserieskmeans_final.py

I ran this in Google Colab's terminal and it worked but let me know if you get any issues

drewm8080 commented:
<img width="567" alt="Screen Shot 2022-11-11 at 5 10 42 PM" src="https://user-images.githubusercontent.com/71193439/201449426-94d5eac2-5dd5-4d63-861f-a4c0faa53e5d.png">


For the most recent file, this was the output. I used a standard scaler on both the x and y axis.

c-myau reviewed (approved):
Some minor tweaks could be needed but otherwise LGTM
